io/zenoh-links/zenoh-link-quic/Cargo.toml

```diff
@@ -54,5 +54,6 @@ zenoh-sync = { workspace = true }
 zenoh-util = { workspace = true }
 # Lock due to quinn not supporting rustls 0.22 yet
 rustls = { version = "0.21", features = ["dangerous_configuration", "quic"] }
-tokio-rustls = "0.24.1"
 rustls-pemfile = { version = "1" }
+tokio-rustls = "0.24.1"
```

Comment:
The dependency should be moved to the workspace.

Reply:
I see there are incompatibility with quinn at the moment. Let's leave it as it is for now and let's update all the dependencies in a separate PR.